jina/serve/networking.py

```diff
@@ -790,21 +855,15 @@ async def send_request_async(
 
     @staticmethod
     def create_async_channel_stub(
```

Comment:
it is awkward for a public method to return a private class' instance

Reply:
made the class public and added necessary docstrings